src/java.desktop/share/classes/java/awt/Component.java

```diff
@@ -2265,7 +2265,7 @@ public void resize(int width, int height) {
      * @param d the dimension specifying the new size
      *          of this component
      * @throws NullPointerException if {@code d} is {@code null}
-     * @see #setSize
+     * @see #getSize
```

Comment:
The old ref seems to be a shurtcut to the `setSize(int, int)` 
take a look to the current spec
https://docs.oracle.com/en/java/javase/19/docs/api/java.desktop/java/awt/Component.html#setSize(java.awt.Dimension)